File: tkpdgm/src/main/java/com/tokopedia/gm/subscribe/data/source/cart/cloud/model/voucher/Attributes.java
package com.tokopedia.gm.subscribe.data.source.cart.cloud.model.voucher;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Attributes {

    @SerializedName("code")
    @Expose
    private String code;
    @SerializedName("detail")
    @Expose
    private String detail;
    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("links")
    @Expose
    private String links;
    @SerializedName("meta")
    @Expose
    private String meta;
    @SerializedName("source")
    @Expose
    private String source;
    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("title")
    @Expose
    private String title;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getDetail() {
        return detail;
    }

    public void setDetail(String detail) {
        this.detail = detail;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getLinks() {
        return links;
    }

    public void setLinks(String links) {
        this.links = links;
    }

    public String getMeta() {
        return meta;
    }

    public void setMeta(String meta) {
        this.meta = meta;
    }

    public String getSource() {
        return source;
    }

    public void setSource(String source) {
        this.source = source;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

}

File: features/communication/attachinvoice/src/main/java/com/tokopedia/attachinvoice/data/Attributes.kt
package com.tokopedia.attachinvoice.data


import com.google.gson.annotations.SerializedName

data class Attributes(
    @SerializedName("Code")
    val code: String = "",
    @SerializedName("CreateTime")
    val createTime: String = "",
    @SerializedName("Description")
    val description: String = "",
    @SerializedName("HrefURL")
    val hrefURL: String = "",
    @SerializedName("Id")
    val id: String = "",
    @SerializedName("ImageURL")
    val imageURL: String = "",
    @SerializedName("Status")
    val status: String = "",
    @SerializedName("StatusId")
    val statusId: String = "0",
    @SerializedName("Title")
    val title: String = "",
    @SerializedName("TotalAmount")
    val totalAmount: String = ""
)